lib/Agrammon/Model.pm6

```diff
@@ -306,7 +306,8 @@ class Agrammon::Model {
     method dump {
         my Str $output;
         for @!evaluation-order.reverse {
-            $output ~= $_.taxonomy ~ "\n";
+            my $level = ($_.taxonomy ~~ m:global/ '::' /).elems;
```

Comment:
This works, but `.taxonomy.comb('::').elems` will too (`comb` being the opposite of `split` - you say what you want).